mandatory/3-function-output.js

```diff
@@ -1,17 +1,21 @@
 // Add comments to explain what this function does. You're meant to use Google!
 function getRandomNumber() {
-  return Math.random() * 10;
+    return Math.random() * 10;
 }
 
 // Add comments to explain what this function does. You're meant to use Google!
 function combine2Words(word1, word2) {
-  return word1.concat(word2);
+    return word1.concat(word2);
 }
 
 function concatenate(firstWord, secondWord, thirdWord) {
-  // Write the body of this function to concatenate three words together.
-  // Look at the test case below to understand what this function is expected to return.
+    // Write the body of this function to concatenate three words together.
+    // Look at the test case below to understand what this function is expected to return.
+    return firstWord + " " + secondWord + " " + thirdWord;
```

Comment:
You could also use the `concat` here. This however combines strings together without spaces between, so you'll have to add them as parameters, for example `firstWord.concat(" ", secondWord, " ", thirdWord)`